src/Elewant/AppBundle/Resources/views/Default/history.html.twig

```diff
@@ -40,29 +41,35 @@
             <div class="row justify-content-md-center">
                 <div class="col col-lg-10">
                     <blockquote class="blockquote">
-                        <p class="text">Did you build this <i>just</i> so that you could use Event Sourcing??
+                        <p class="text">
+                            {% trans %}fifth-paragraph{% endtrans %}
                             <span class="blockquote-footer">Jeroen van der Gulik (@n0x13)</span>
                         </p>
                     </blockquote>
-                    <p class="text">The answer, as with many things, is "it's complicated". But we really hope that by
-                        building and using this system together, we can learn and experience what it's like to have an
-                        event sourced project in production. A project that serves a purpose beyond classroom examples.
+                    <p class="text">
+                        {% trans %}sixth-paragraph{% endtrans %}
                     </p>
-                    <p class="text">Of course, it would not hurt if this project persuades someone to join us at
-                        <a href="http://future500.nl/vacancy/">Future500</a>...
+                    <p class="text">
+                        {% trans with {'%link%': '<a href="http://future500.nl/vacancy/">Future500</a>'} %}
+                            seventh-paragraph
+                        {% endtrans %}
                     </p>
                     <p class="text">
-                        So enjoy, and don't forget to
                         {% if not is_granted('IS_AUTHENTICATED_REMEMBERED') %}
-                            <a href="{{ path('hwi_oauth_twitter_login') }}">Join the herd!</a>
+                            {% trans with {'%link%': '<a href="' ~ path('hwi_oauth_twitter_login') ~ '">' ~ ('site.join-the-herd'|trans) ~ '</a>' } %}
+                                eight-paragraph
+                            {% endtrans %}
                         {% else %}
-                            <a href="{{ path('herd_tending') }}">Tend to your herd!</a>
+                            {% trans with {'%link%': '<a href="' ~ path('herd_tending') ~ '">' ~ ('site.tend-to-your-herd'|trans) ~ '</a>' } %}
+                            eight-paragraph
+                            {% endtrans %}
                         {% endif %}
                     </p>
                     <p class="text">
-                        Kind regards,<br />
-                        <br />
-                        Elena, Jasper, Mitchel, Mark and Ramon
+                        {{ 'kind-regards'|trans }}
+                    </p>
+                    <p class="text">
+                        Elena, Jasper, Mitchel, Mark && Ramon
```

Comment:
`&&` should be `&amp;` I guess?

Reply:
No, it was a bit of a programmers joke. Failed miserably :)

Reply:
And I failed miserably getting the joke 😞 

So... `&amp;&amp;` then? 😄

Reply:
Ha! Yes, you're absolutely right, I failed at missing the actual point :D